packages/terra-horizontal-divider/src/HorizontalDivider.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './HorizontalDivider.scss';
+
+const cx = classNames.bind(styles);
+
+const HorizontalDivider = ({ ...customProps }) => {
+  const HorizontalDividerClassNames = cx([
+    'divider',
+    customProps.className,
+  ]);
+
+  return (<hr {...customProps} className={HorizontalDividerClassNames} />);
```

Comment:
This is up to your preference, the surrounding parentheses are not needed when the return is a single line. 
```
return <hr {...customProps} className={HorizontalDividerClassNames} />;
```

Reply:
Updated https://github.com/cerner/terra-core/pull/962/commits/5fb27563217e211a5ec708f6dc65da66fb369f31#diff-25e37f5cb806bb82d51633a0117496a5R13